src/components/Header/Header.jsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import React, { useState } from "react";
+// import { Link } from 'react-router-dom';
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
+    mobileMenu ? setMobileMenu(false) :
+   setMobileMenu(true);
+  };
   return (
     <header className={styles.header}>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
       <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
+        <ul className={ mobileMenu ? '' :  styles['hide-mobile-menu']}>
+          <img src={icon_close} alt="" className={styles.icon_close} onClick={toggleMenu} />
+         <li>home</li>
```

Comment:
```suggestion
 const navBarOptions = ["home", "about"];
        {navBarOptions.map((option) => {
            return (
              <li key={option}>
                <Typography elType="body3">{option}</Typography>
              </li>
            );
          })}
```

Reply:
We can add eltype body3 with the following details
`.body3 {
  font-size: 1.12rem;
  line-height: 1.75rem;
}`

Reply:
now its 2, may be some more in future, we make an array of route name and rendered component and iterate over it in the JSX code

Reply:
Please re-add the `Link` components to the nav bar—right now it doesn't navigate.
`<Link to="/">home</Link>`
`<Link to="/about">About</Link>`